content/en/docs/how-to/manage-signatures.md

```diff
@@ -0,0 +1,119 @@
+---
+title: "Manage signatures"
+description: "How to list, inspect, and delete a signature on an artifact"
+type: docs
+weight: 10
+---
+
+{{% alert title="Important" color="important" %}}
+When managing signatures on artifacts in remote repositories that require authentication, you must authenticate using `notation login` before you can manage signatures on artifacts in those repositories. For more information, see [Authenticate to a remote repository]({{< ref "/docs/how-to/registry-authentication" >}}).
+{{% /alert %}}
+
+## List signatures on an artifact
+
+Use `notation list` to show the signatures associated with an artifact.
+
+{{% alert title="Important" color="important" %}}
+When using `notation list`, use the fully qualified artifact reference, including the digest. For example, `localhost:5001/net-monitor@sha256:111222333444555666777888999000aaabbbcccdddeeefff0001112223334445`. You can use `notation list` with a tag, for example, `localhost:5001/net-monitor:latest`. Using a tag is not recommended since a tag is mutable and may be updated to point to a different version of the artifact. A digest is not mutable and always points to the same version of the artifact.
+
+To find the digest of an artifact, use `notation list` with the artifact reference. For example, `notation list localhost:5001/net-monitor:v1`. You can also use `oras manifest fetch --descriptor` on remote registries that do not require authentication. For example, `oras manifest fetch --descriptor REGISTRY/IMAGE:TAG`.
```

Comment:
```suggestion
To find the digest of an artifact, use `notation list` with the artifact reference. For example, `notation list localhost:5001/net-monitor:v1`. You can also use `oras manifest fetch --descriptor` on OCI registries. For example, `oras manifest fetch --descriptor REGISTRY/IMAGE:TAG`.
```

Reply:
ORAS supports all OCI-compatible registries but it also requires to authenticate with the registries before using ORAS.

Reply:
thanks @FeynmanZhou 

fixed